[New documentation] Redirect (#656)
Sylius/SyliusResourceBundle: docs/redirect.md

# Redirect

After that an action has been performed, the operation can be redirected to another operation.

<!-- TOC -->
* [Default redirections](#default-redirections)
* [Custom redirection](#custom-redirection)
* [Pass arguments to your redirection](#pass-arguments-to-your-redirection)
<!-- TOC -->

## Default redirections

Redirections are configured on your operations with these default behaviours.

| Operation   | Redirection                         |
|-------------|-------------------------------------|
| create      | `show` if exists, otherwise `index` |
| update      | `show` if exists, otherwise `index` |  
| delete      | `index`                             |
| bulk_delete | `index`                             |

## Custom redirection

> As an example, let's configure a custom redirection fo create & update operations.

Reviewer: ```suggestion
For example, let's configure a custom redirection to create & update operations.
```
